mama_ng_contentstore.env

```diff
@@ -0,0 +1,10 @@
+# Add Environment Variables
+
+
+SECRET_KEY=6(16ds+i2+%ik6z&!yer+ga9m=e%jcqiz_5wszg)r-z!2--b1d
```

Comment:
You probabl want to replace these when running on the server, since this is a public repo.